src/reporting/ReputationToken.sol

```diff
@@ -36,7 +43,7 @@ contract ReputationToken is DelegationTarget, StandardToken {
         symbol = "REP";
         decimals = 18;
         // FIXME: DELETE THIS BEFORE LAUNCH
-        var _reputationFaucet = controller.lookup("reputationFaucet");
+        var _reputationFaucet = controller.lookup("ReputationFaucet");
```

Comment:
Heh, so apparently we do use the `ReputationFaucet`, even though it doesn't actually faucet reputation.  :P